### DISCUSSION TITLE
Suggestions for Ollama Wiki Modifications

### DISCUSSION THREAD

**Discussion by KimCookieYa** [TheR1D/shell_gpt#654] [Show and tell]
I wish I could add the following description to [ShellGPT configuration] section. This will help others.

---

## ShellGPT configuration

[](https://github.com/TheR1D/shell_gpt/wiki/Ollama#shellgpt-configuration)

Now when we have Ollama backend running we need to configure ShellGPT to use it. To communicate with local LLM backends, ShellGPT utilizes [LiteLLM](https://github.com/BerriAI/litellm). To install it run:

```shell
pip install "shell-gpt[litellm]"
```

Edit `~/.config/shell_gpt/.sgptrc` to specify a default model.

```shell
vim ~/.config/shell_gpt/.sgptrc
```

- DEFAULT_MODEL=ollama/your_local_model
- OPENAI_USE_FUNCTIONS=false
- USE_LITELLM=true
- OPENAI_API_KEY=any_value

**Comment by hesreallyhim** [comment]
Came here to say this!!! YES please update this. the ollama integration is working great once i fixed this

**Comment by pcause** [comment]
did that and set the base url to what ollama uses and didn't work.  got a read error.  tried various endpoints for ollama and nothing worked.
